### ISSUE TITLE
Add reply from notification

### ISSUE THREAD

**Issue by mimi89999** [zom/Zom-Android-XMPP#211]
That is a nice Android N feature

**Comment by n8fr8** [comment]
Agreed! That said, most of our target users in Asia are on L and M... but still let's find ways to support the new cool stuff on N and O simultaneously.
